Etabs_Adapter/Types/Comparer.cs

```diff
@@ -39,42 +39,24 @@ public partial class ETABS17Adapter : BHoMAdapter
 #else
     public partial class ETABS2016Adapter : BHoMAdapter
 #endif
-    { 
-
-    /***************************************************/
-    /**** BHoM Adapter Interface                    ****/
-    /***************************************************/
-
-        protected override IEqualityComparer<T> Comparer<T>()
-        {
-            Type type = typeof(T);
-
-            if (m_Comparers.ContainsKey(type))
-            {
-                return m_Comparers[type] as IEqualityComparer<T>;
-            }
-            else
-            {
-                return EqualityComparer<T>.Default;
-            }
-
-        }
-
+    {
 
         /***************************************************/
-        /**** Private Fields                            ****/
+        /**** BHoM Adapter Interface                    ****/
         /***************************************************/
 
-        private static Dictionary<Type, object> m_Comparers = new Dictionary<Type, object>
+        protected void SetupComparers()
         {
-            {typeof(Node), new BH.Engine.Structure.NodeDistanceComparer(3) },
-            {typeof(ISectionProperty), new BHoMObjectNameOrToStringComparer() },
-            {typeof(IMaterialFragment), new BHoMObjectNameComparer() },
-            {typeof(ISurfaceProperty), new BHoMObjectNameComparer() },
-            {typeof(BH.oM.Adapters.ETABS.Elements.Diaphragm), new BHoMObjectNameComparer() },
-        };
-
-
+            AdapterComparers = new Dictionary<Type, object>
```

Comment:
Should be moved to the Adapters constructor as a property. [Follow the example here](https://github.com/BHoM/GSA_Toolkit/blob/aaccbf32498e8f45a85dc447bc3d0984dd93fba3/GSA_Adapter/GSAAdapter.cs#L59-L68)

Reply:
As above, keeping out of constructor for findability, discussed with @alelom .